components/Cards/SchoolCard/__tests__/SchoolCard.test.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import createShallowSnapshotTest from 'test-utils/createShallowSnapshotTest';
+import SchoolCard from '../SchoolCard';
+
+const locations = [
+  {
+    va_accepted: true,
+    address1: '825 Battery Street',
+    address2: '3rd Floor',
+    city: 'San Francisco',
+    zip: '94111',
+  },
+];
+
+describe('SchoolCard', () => {
+  it('should render with required props', () => {
+    createShallowSnapshotTest(
+      <SchoolCard
+        hasHardwareIncluded
+        hasHousing
+        hasOnline
+        hasOnlyOnline={false}
+        isFullTime
+        locations={locations}
+        logoSource="logoSource.com"
+        name="school name"
+        website="website.com"
+      >
+        <p>Testing!</p>
+      </SchoolCard>,
+    );
+  });
+  it('call to showBackOfCard/showFrontOfTheCard flips isFrontOfCardShowing', () => {
+    const wrapper = shallow(
+      <SchoolCard
+        cardFlipCallback={() => {}}
+        hasHardwareIncluded
+        hasHousing
+        hasOnline
+        hasOnlyOnline={false}
+        isFullTime
+        locations={locations}
+        logoSource="source"
+        name="school name"
+        website="website"
+      >
+        Test
+      </SchoolCard>,
+    );
+    const instance = wrapper.instance();
+    wrapper.setState({ isFrontOfCardShowing: true });
```

Comment:
line 52 and 53 are unnecessary.

I'd also love if you tested content existing AND the state variable... So like check for the existence of content that exists only on the front card when `isFrontOfCardShowing` is true

Reply:
Updated tests here